roles/nuage_master/tasks/main.yaml

```diff
@@ -33,6 +33,14 @@
 
 - include: certificates.yml
 
+- name: Install Nuage VSD user certificate
+  become: yes
+  copy: src="{{ vsd_user_cert_file }}" dest="{{ cert_output_dir }}/{{ vsd_user_cert_file | basename }}"
```

Comment:
vsd_user_cert_file and vsd_user_key_file are parameters that will be populated in the ansible "nodes" parameter file that will be used at the time of running playbooks.